src/libraries/System.Private.Xml/src/System/Xml/Xsl/Runtime/ContentIterators.cs

```diff
@@ -1,6 +1,7 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+#nullable disable
```

Comment:
Why is this disabled?

Reply:
XSLT runtime is only used by generated code and won't help customers as none of these APIs is meant for consumption so not worth annotating (in long term perhaps to find some NREs but we got limited time now so this part was cut)